src/sqlancer/stonedb/gen/StoneDBTableDeleteGenerator.java

```diff
@@ -37,23 +39,25 @@ public SQLQueryAdapter getQuery() {
         }
         sb.append(" FROM ");
         sb.append(randomTable.getName());
-        if (Randomly.getBoolean()) {
-            sb.append(" AS ");
-            sb.append(globalState.getSchema().getRandomTable().getName());
-        }
+        // do not need AS if does not use tbl_alias later
+        // if (Randomly.getBooleanWithSmallProbability()) {
```

Comment:
remove commented lines ( if you will need this in the future then keep it here should be fine for now)